src/features/pricing/additionalServicePricing/__tests__/AdditionalServicePricing.test.tsx

```diff
@@ -27,13 +30,4 @@ describe('AdditionalServicePricing', () => {
 
     expect(wrapper.html()).toMatchSnapshot();
```

Comment:
Can you switch this and the other changed snapshots into `wrapper.render()`?

Reply:
Yup, changed them.